vercel.json

```diff
@@ -263,6 +263,22 @@
     {
       "source": "/inspector/sdk/web/",
       "destination": "/docs/implementation/inspector/sdk/web"
+    },
+    {
+      "source": "/data-design/import-tracking-plan/",
+      "destination": "/docs/workspace/tracking-plan/importing"
+    },
+    {
+      "source": "/data-design/import-tracking-plan",
+      "destination": "/docs/workspace/tracking-plan/importing"
+    },
+    {
+      "source": "/data-design/import-tracking-plan/",
```

Comment:
Is it same source listed twice with 2 different destinations? How does it work?

Reply:
Good catch! That's a copy-paste error. Fixed in 1fd5ed4b09987484fbe45b5c0bf4ad3c24ef16af